n3/n3-tests.ts

```diff
@@ -0,0 +1,204 @@
+/// <reference path="./index.d.ts" />
+/// <reference path="../node/index.d.ts" />
+
+
+/**
```

Comment:
Do I need `/// <reference types="node" />`?
I make use of `fs` and `stream` from nodejs.